r2i/tflite/Makefile.am

```diff
@@ -43,6 +43,9 @@ libtflite_la_LIBADD =                \
     $(TFLITE_LIBS)           \
     $(CODE_COVERAGE_LIBS)
 
+libtflite_la_LDFLAGS =		     \
```

Comment:
Can you make sure that all `\` are in the same column in the file?

Reply:
Agree